admin_guide/overcommit.adoc

```diff
@@ -197,6 +197,64 @@ quota.openshift.io/cluster-resource-override-enabled: "false"
 In an overcommitted environment, it is important to properly configure your node
 to provide best system behavior.
 
+[[configuring-reserve-resources]]
+=== Reserving memory across quality of service tiers
+
+{product-title} can be configured to use xref:../../security/hosts_multitenancy.adoc#security-hosts[control groups (cgroups) to enforce QoS.
+
+Cgroups limit, account for, and isolate the resource usage (CPU, memory, disk I/O, network, etc.)
+to ensure that containers on the same host are not impacted by each other.
```

Comment:
I'm not sure I'd mention cgroups at all.  One can use `experimental-qos-reserved` without understanding how the limits are enforced, and I'm not sure this description of cgroups adds value.